### DISCUSSION TITLE
Survey 2016

### DISCUSSION THREAD

**Discussion by felipefialho** [frontendbr/forum#224] [Discussão]
Fala pessoal,

Em 2014, fiz um Survey e consolidei os [resultados aqui](http://www.felipefialho.com/survey/2014)

Dois anos é um tempo muito bom para rolar outra pesquisa e assim será feito. 

Será que muita coisa mudou? Vamos descobrir! 

[Aqui o link da pesquisa](https://lfeh.typeform.com/to/LD3mRL)

Respondam e compartilhem para que a pesquisa atinja o maior número possível de desenvolvedores.

Nóis.

**Comment by lnfnunes** [comment]
Boa!

**Comment by renatogalvones** [comment]
:metal: Done

**Comment by MaxWilliamJF** [comment]
:metal: Done ²

**Comment by felipefialho** [comment]
Se puderem ajudar a divulgar, agradeço ;)

**Comment by Kirmayrtomaz** [comment]
Done XD

**Comment by hevertoncastro** [comment]
Respondido e divulgado!

**Comment by caiofsouza** [comment]
Done :D

**Comment by dleitee** [comment]
Done :smile:

**Comment by andre-mendes** [comment]
foi :dancer:

**Comment by iamgbayer** [comment]
respondido!

**Comment by ramonsenadev** [comment]
Done! :)

**Comment by napalmdev** [comment]
Done !!!

**Comment by tcelestino** [comment]
👍

**Comment by rubenmarcus** [comment]
Cara vi que o Tableless fez uns tempos atrás tambem. Muito bom essa! Mas é dinamico? se eu responder agora atualiza os dados? Respondido

**Comment by felipefialho** [comment]
Não, vou consolidar daqui uns meses :)

**Comment by marceloogeda** [comment]
Done!

**Comment by dsoldera** [comment]
Done!

**Comment by notnotgabriel** [comment]
respondido :metal:

**Comment by felipefialho** [comment]
E ai está o consolidado pessoal ;)

http://www.felipefialho.com/survey/
